components/screens/full-pages/ledger/ImportLedgerStart.tsx

```diff
@@ -0,0 +1,59 @@
+import Header from "@/components/GeneralHeader";
+import { useNavigate } from "react-router-dom";
+
+export default function ImportLedgerStart() {
+  const navigate = useNavigate();
+
+  const steps = [
```

Comment:
It's just a recommendation, but mixing types in an collection, in my experience, has most of the time proven to be mistake.

I would rather see a `const isChecked = number === 4` or something like `array.indexOf(elem) === array.length - 1`

Since arrays keep their order you could leverage the index of the map iteration as number.

Reply:
https://github.com/forbole/kastle/pull/41/commits/3571a4f1aed4848ec1588f9b758f21e25f0ec41f